pmd-plsql/etc/grammar/PldocAST.jjt

```diff
@@ -1186,6 +1186,28 @@ void RestOfStatement() #void :
 
     [ OrderByClause() ]
     [ RowLimitingClause() ]
+    [ ForUpdateClause() ]
+}
+
+/**
+ * @see https://docs.oracle.com/database/121/SQLRF/statements_10002.htm#i2126016
+ */
+void ForUpdateClause() #void :
```

Comment:
I think, this node should not be #void - otherwise we can't see, that the select statement had indeed such a for-update-clause.